portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,75 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
-#greeting-container {
-  margin-top: 20px;
-}
+:root {
+  --WOODSMOKE: #222629;
+  --EBONYCLAY: #474B4F;
+  --NEVADA: #61666a;
+  --SILVERSAND: #C5C6C7;
+  --AQUAMARINE: #66FCF1;
+  --KEPPEL: #45A29E;
+}
+* {
+  box-sizing: border-box;
+}
+html {
+  height: 100%;
+}
+body {
+  margin: 0;
+  height: 100%;
+  font-family: 'Montserrat', sans-serif;
+  background-color: var(--WOODSMOKE);
+  color: var(--SILVERSAND);
+}
+
+.nav {
+  display: flex;
+  background-color: var(--NEVADA);
+  padding: 16px 0;
+}
+.header {
+  background-color: var(--EBONYCLAY);
+  text-align: center;
+}
+.header__title {

```

Comment:
I've not seen the convention of using two double underscores, but it would be fine as long as you are comfortable with it and make it consistent.

Reply:
Oh, the double underscores is [CSS custom property](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) syntax, added in CSS3